src/test/resources/testagency/agency.txt

```diff
@@ -0,0 +1,2 @@
+agency_id,agency_name,agency_url,agency_timezone
+agency,Fake Agency,http://fake.example.com,America/New_York
```

Comment:
I unzipped the "testagency" . OTP2 is capable of reading input data from a catalog, it is not a required to be a zip file. Having the input data attached as files make the maintenance of these files simpler.